tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -521,7 +521,6 @@ def test_pantsd_sigint(self):
       signal.SIGINT,
       regexps=["""\
 \\[INFO\\] Sending SIGINT to pantsd-runner with pid [0-9]+, waiting up to 5\\.0 seconds before sending SIGKILL\\.\\.\\.
-Interrupted by user.
 Interrupted by user:
 Interrupted by user over pailgun client!
```

Comment:
Having all these very similar messages in a row looks pretty silly, but I didn't spend time to fix it up in #6574. Sorry!